doc/user-guide/ICE_tutorial.md

```diff
@@ -61,9 +61,9 @@ In order to enable signalling we need an instance of [MongooseIM] running with t
 You can find MongooseIM installation instructions on [this page](Getting-started.md).
 Once you have cloned the repository and compiled the project, you need to modify the `mongooseim.cfg` config file (you can find this file at `$REPO/_build/prod/rel/mongooseim/etc/mongooseim.cfg`, where `$REPO` is a top-level directory of the cloned repo).
 You can use [this](ICE_tutorial/mongooseim.cfg) configuration file and modify the relevant part:
```

Comment:
This file is in the `doc/user-guide/ICE_tutorial/` directory and seems old and is not TOML for sure, so the modification described below will not work. I think we should either provide a new file or just send the user to the default that MIM contains in `rel/files`?

Reply:
I will remove this sentence from line 63 and I already created a ticket to provide the user with sample TOML config.